### DISCUSSION TITLE
Node.js version 16.x deployments created on or after 2023-08-15 will fail to build

### DISCUSSION THREAD

**Discussion by ardmoreuk** [vercel/community#2944] [Help] [answered]
### Summary

Hello folks,
Let's say I am using Node 16. 
My last build was on the 1st of June.
Will it still work with Node 16 as expected even after 2023-08-15, unless I will redeploy my app?

### Steps to Reproduce

n/a

**Comment by amyegan** [comment, marked as answer]
Hi @ardmoreuk! Existing deployments with Serverless Functions will not be affected, but future deployments will show an error if the Node.js version is not upgraded. Although existing deployments will continue to work, we recommend upgrading as soon as possible to ensure you receive security updates. You can learn more here: https://vercel.com/changelog/node-js-14-and-16-are-being-deprecated

**Reply by ardmoreuk** [reply]
Thank you, Amy :)

**Reply by devfive** [reply]
> Vercel is announcing the deprecation of Node.js 14 and 16, which will be discontinued on August 15th 2023 and February 6th 2024 respectively.

@amyegan shouldn't Node 16 work until February 6th 2024?

**Reply by amyegan** [reply]
It will, but it was updated from the original date of 2023-08-15 which was accurate at the time of the older post here. Thanks for pointing that out for anyone else who comes across this discussion!
